Makefile

```diff
@@ -6,7 +6,7 @@ build:
 .PHONY: build
 
 fmt:
-	ocamlformat --inplace --enable-outside-detected-project $(OCAML_SRCFILES)
+	esy x ocamlformat --inplace --enable-outside-detected-project $(OCAML_SRCFILES)
```

Comment:
I don't use esy to develop the extension so let's leave this target useful for everyone.

It's just easy to run make inside of esy x.